### ISSUE TITLE
Livewire error after clearing search

### ISSUE THREAD

**Issue by margarizaldi** [laravel-filament/demo#52]
Step to reproduce:

- Go filament demo, choose whatever page from sidebar (products, orders, etc).
- As error doesn't always happened, try to search: 576674888648395525 or similar to ensure you catch the error.
- Refresh the page (with the table search url query still there)
- Clear or modify the search bar and an error will pop up (please check the error log, my app showed a livewire error)

https://user-images.githubusercontent.com/26832856/194875421-658223bb-101b-4785-b8a9-61aa7c6019c7.mp4

**Comment by zepfietje** [comment]
Hey @margarizaldi, thanks for reporting! Do you think this is a Filament issue that should be filed at https://github.com/filamentphp/filament?

**Comment by margarizaldi** [comment]
Hi @zepfietje I was thinking to report this issue to filament repo, but @danharrin suggested me to go here. This is actually a filament issue.

--- Another case, maybe useful:
I also remember I got this exact same error several times when I use long series of numbers (an order id from API fetch, not incrementing) as primary key.

It happened when I clicked create -> cancel -> create, and also create -> submit (modal action, simple resource).

But it was happened not so consistent so I failed to replicate the issue and just changing my table schema to make another increment int as primary key and set unique index for that long order id column, the error is gone and never think about that again.
